<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameWindow</class>
 <widget class="QWidget" name="gameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>4</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Правила</string>
  </property>
  <widget class="QPushButton" name="prevPageDescBTN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>30</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>470</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Заголовок&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>470</width>
     <height>190</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Описание&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPageDescBTN">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>30</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-100</y>
     <width>620</width>
     <height>630</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="capitalButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Капитал</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agresssionButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Агрессия</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mediaButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>СМИ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blatButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Блат</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionCard">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>470</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Описание&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleCard">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>470</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Заголовок&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPageCardBTN">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>30</width>
     <height>230</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevPageCardBTN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>30</width>
     <height>230</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>550</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-----------------------------------------------------------------------------------------------------------------------------------------</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>prevPageDescBTN</zorder>
  <zorder>title</zorder>
  <zorder>description</zorder>
  <zorder>nextPageDescBTN</zorder>
  <zorder>capitalButton</zorder>
  <zorder>agresssionButton</zorder>
  <zorder>mediaButton</zorder>
  <zorder>blatButton</zorder>
  <zorder>descriptionCard</zorder>
  <zorder>titleCard</zorder>
  <zorder>nextPageCardBTN</zorder>
  <zorder>prevPageCardBTN</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
